### ISSUE TITLE
Bug: Csets with 0 concepts

### ISSUE THREAD

**Issue by joeflack4** [jhu-bids/TermHub#241]
### Overview
Csets with 0 concepts are appearing in the search.

### To Reproduce
1. Go to cset search adn click the dropdown

### Screenshots
<img width="1792" alt="search broken" src="https://user-images.githubusercontent.com/13045020/217589489-191319e1-2121-4348-9fab-d0febbe57fe0.png">

**Comment by hugwuoke** [comment]
Interesting, I think Siggie has been going about this by trying to make sure loading empty Csets doesn't cause an error while this suggests we shouldn't be using them at all.

**Comment by joeflack4** [comment]
Dang, I did a bad job of creating this issue. Firstly looks like my screenshot doesn't actually show the issue. And secondly, I'm not sure if I described why this is a problem. I don't remember, but it may have been the case that there were concepts that actually had non-zero concepts but were showing as having 0 concepts, but that is probably fixed, and you're also correct that Siggie is handling this now. Thanks for closing!
